doc/ext/autodoc.rst

```diff
@@ -376,6 +376,13 @@ There are also new config values that you can set:
    .. versionadded:: 1.3
 
 
+.. confval:: autodoc_signature_as_is
+
+   If set ``True``, the function or method signatures are shown in docstrings
+   as is (for example, the constant names are still the constant names, not
+   their values).
```

Comment:
`autodoc_docstring_signature` and `autodoc_signature_as_is`, which is priority?

It might be better to integrate source of signature setting:
```
autodoc_signature_sources = ['docstring', 'source_file', 'inspection']  # source_file equals as-is
```